db/models.py

```diff
@@ -50,3 +58,70 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(to=User, on_delete=models.CASCADE)
```

Comment:
It's recommended to use settings.AUTH_USER_MODEL instead of User.
Also provide `related_name`